conda/core/prefix_data.py

```diff
@@ -470,6 +467,7 @@ def _load_single_record(self, prefix_record_json_path):
     # region Python records
 
     @property
+    @deprecated("25.9", "26.3", addendum="Use PrefixData.get('python').")
```

Comment:
any downsides to changing the body here to do that?

```python
@property
@deprecated("25.9", "26.3", addendum="Use PrefixData.get('python').")
def _python_pkg_record(self):
    """Return the prefix record for the package python."""
    return self.get("python")
```

Reply:
Since it's marked for deprecation, I'd rather not touch it.